docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1612,6 +1612,154 @@ Note/limitations:
 
 - Applying a uniform non-response rate limits the impact of race/ethnicity, age, and sex to affect the sampled population. This might make some aspects of PRL easier as it is less likely the same simulants will be missing from each sample.
 
+Women, Infants, and Children (WIC)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+WIC is a government benefits program designed to support mothers and young 
+children. The main qualifications are income and presence of young children 
+in the home. 
+
+
+**When to Sample** 
+
+- Sample compiled on the time step containing Jan 1st of each year (the time step might end on Jan 2nd, Jan 15th, Jan 27th, etc.)
+
+**What to Sample** 
+
+.. list-table:: Simulant Attribute to Sample 
+  :widths: 20
+  :header-rows: 0
+
+  * - Household number (random indicator which is the same for simulants in the same household)
+  * - Unique simulant ID (for PRL tracking)
+  * - First name
+  * - Middle initial 
+  * - Last name
+  * - Age 
+  * - DOB 
+  * - Home Address 
+  * - Home Zip Code 
+  * - Race/ethnicity 
+
+Here is an example: 
+
+.. image:: WIC_example.png
+
+Note: for household number, if a tracked mother/guardian is in the sim along with their 
```

Comment:
I had been imagining that the "household number" or "household ID" could be the same as the "address ID" (since I believe we're going to stop generating addresses in the simulation and have a unique identifier for each address, adding the addresses in the first post-processing/observer step).

But it sounds like what you're saying here is that the household ID cannot necessarily be the same as the address ID -- it sounds like they could be the same thing *except* in the case where the mother and child live at different addresses, in which case a new ID would need to be generated. Is that correct?

Also, does "tracked mother/guardian" here mean tracked by the sim, or tracked by the WIC observer?